src/main/java/com/metamx/common/parsers/JSONPathParser.java

```diff
@@ -225,7 +235,7 @@ private boolean isFlatList(List<Object> list)
 
   /**
    * Specifies a field to be added to the parsed object Map, using JsonPath notation.
-   *
+   * <p/>
```

Comment:
sorry, IntelliJ does that automatically when I hit the formatter. Any idea how to turn that off?